schema/partner_show.js

```diff
@@ -98,6 +98,26 @@ const PartnerShowType = new GraphQLObjectType({
         gravity(`partner/${partner.id}/show/${id}`)
           .then(({ events }) => events),
     },
+    counts: {
+      type: new GraphQLObjectType({
+        name: 'PartnerShowCounts',
+        fields: {
+          artworks: {
+            type: GraphQLInt,
+            args: {
+              artist_id: {
+                type: GraphQLString,
+                description: 'The slug or ID of an artist in the show.',
+              },
```

Comment:
@dzucconi I was unsure how to make `numeral` work with this field that takes an optional argument. Is this a requirement for getting this merged or is it something that can be added when needed?